db/seeds.rb

```diff
@@ -1,13 +1,16 @@
-# We have attempted to make seeding flexible depending on the environment the seed is occuring. This allows us to,
-# for example add dummy records into a User Testing or staging environment that we wouldn't want in development. This
-# works by having a file for each environment within db/seeds which handles anything specific to that environment.
-# Anything that is mutual to all goes in db/seeds/all.rb. db/seeds.rb then simply becomes a function that calls all.rb
-# and the relevant environment specific file. This solution and the code below were taken from
-# https://archive.dennisreimann.de/blog/seeds-for-different-environments/
-['all', Rails.env].each do |seed|
-  seed_file = FloodRiskEngine::Engine.root.join('db', 'seeds', "#{seed}.rb")
-  if File.exist?(seed_file)
-    puts "FRE:SEED MSG: Loading #{seed} data"
-    require seed_file
-  end
-end
+require 'dibber'
+# https://github.com/reggieb/Dibber
+Seeder = Dibber::Seeder
+Seeder.seeds_path = File.expand_path('seeds', File.dirname(__FILE__))
+
+# Note that if an Exemptions exists seeding will just update the summary
+Seeder.new(
+  FloodRiskEngine::Exemption,
+  'flood-risk-engine/exemptions.yml',
+  name_method: :code,
+  attributes_method: :summary
+).build
+
+puts Seeder.report
+
+
```

Comment:
Looks a bit nicer to me and doesn't lean on I18n which was maybe a bit confusing before. @autotelik this came out of trying to seed Heroku (env = production) so had to find a way to seed without just relying on Rails.env. Event in live env we will need to run some initial seeds to set up database. Checking presence of record first prevents it being added again. However I think I'd feel safer if we didn't attempt seeding Exemptions at all if any Exemptions exist?